modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -143,7 +142,7 @@
 
 /datum/action/cooldown/borer/evolution_tree/Trigger(trigger_flags)
 	. = ..()
-	if(!.)
+	if(. == FALSE)
```

Comment:
The purpose of using `!.` instead of `. == FALSE` is that `!.` catches empty `return` statements too.
```suggestion
	if(!.)
```